20231027	19:13:27	5b2b8c80-418c-4143-a15a-767e54994a53	role "accel3" does not exist: error	error: role "accel3" does not exist
    at Parser.parseErrorMessage (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:369:69)
    at Parser.handlePacket (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:188:21)
    at Parser.parse (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:103:30)
    at Socket.<anonymous> (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/index.ts:7:48)
    at Socket.emit (node:events:514:28)
    at Socket.emit (node:domain:488:12)
    at addChunk (node:internal/streams/readable:376:12)
    at readableAddChunk (node:internal/streams/readable:349:9)
    at Socket.Readable.push (node:internal/streams/readable:286:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)	
20231027	19:14:20	176ac74d-3389-48d6-8555-a432b37f35d1	database "accel3" does not exist: error	error: database "accel3" does not exist
    at Parser.parseErrorMessage (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:369:69)
    at Parser.handlePacket (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:188:21)
    at Parser.parse (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:103:30)
    at Socket.<anonymous> (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/index.ts:7:48)
    at Socket.emit (node:events:514:28)
    at Socket.emit (node:domain:488:12)
    at addChunk (node:internal/streams/readable:376:12)
    at readableAddChunk (node:internal/streams/readable:349:9)
    at Socket.Readable.push (node:internal/streams/readable:286:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)	
20231027	19:17:28	87157bf0-4fca-4430-b34c-282383a48e30	database "accel3" does not exist: error	error: database "accel3" does not exist
    at Parser.parseErrorMessage (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:369:69)
    at Parser.handlePacket (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:188:21)
    at Parser.parse (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:103:30)
    at Socket.<anonymous> (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/index.ts:7:48)
    at Socket.emit (node:events:514:28)
    at Socket.emit (node:domain:488:12)
    at addChunk (node:internal/streams/readable:376:12)
    at readableAddChunk (node:internal/streams/readable:349:9)
    at Socket.Readable.push (node:internal/streams/readable:286:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)	
20231027	19:18:18	b5354fb2-6cbf-4cbc-9f8f-25fa79ec6e71	database "accel3" does not exist: error	error: database "accel3" does not exist
    at Parser.parseErrorMessage (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:369:69)
    at Parser.handlePacket (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:188:21)
    at Parser.parse (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:103:30)
    at Socket.<anonymous> (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/index.ts:7:48)
    at Socket.emit (node:events:514:28)
    at Socket.emit (node:domain:488:12)
    at addChunk (node:internal/streams/readable:376:12)
    at readableAddChunk (node:internal/streams/readable:349:9)
    at Socket.Readable.push (node:internal/streams/readable:286:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)	
20231027	19:19:25	76562689-a5da-4910-9337-15327c58bef1	database "accel3" does not exist: error	error: database "accel3" does not exist
    at Parser.parseErrorMessage (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:369:69)
    at Parser.handlePacket (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:188:21)
    at Parser.parse (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/parser.ts:103:30)
    at Socket.<anonymous> (/Users/accel33/Code/delfosti/accel-maeshiro/node_modules/pg-protocol/src/index.ts:7:48)
    at Socket.emit (node:events:514:28)
    at Socket.emit (node:domain:488:12)
    at addChunk (node:internal/streams/readable:376:12)
    at readableAddChunk (node:internal/streams/readable:349:9)
    at Socket.Readable.push (node:internal/streams/readable:286:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)	
20231028	12:15:50	66b01ae0-54be-4b24-99e5-8f4b289c400a	classesAndStrings.filter is not a function: TypeError	
20231028	12:53:16	88adaf62-414a-4a89-9c0a-78e53fd6ad26	invalid input value for enum usuarios_rol_enum: "Delivery": QueryFailedError	
20231028	12:53:27	6b615398-eab7-46d4-973a-f47eef1ea816	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	12:53:49	4d372d29-49ac-4e14-99aa-ec704f833e42	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	12:56:25	b0db8303-eac4-426d-8085-21ba30852461	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	12:56:35	5acbdbc5-63c0-41b4-a232-89112877d523	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	12:56:40	0895f73e-9339-488c-a53d-9d62e695616d	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	12:57:00	1a3827a3-3e77-4594-aaef-6441f576c187	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	13:08:09	c4caf742-f56d-4417-af0c-db8dc155a165	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	13:09:45	50e05303-5cdc-4d27-8189-f11cce8965a8	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	15:40:49	697ca156-fa5f-41f5-a15f-2101571adefe	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	16:57:29	c8450f6b-43ca-4ea1-9845-8ee5a71e1edd	invalid input value for enum usuarios_rol_enum: "Delivery": QueryFailedError	
20231028	16:58:29	230b70a8-2023-46ad-86c4-6c431288b748	invalid input value for enum usuarios_rol_enum: "Delivery": QueryFailedError	
20231028	17:01:29	178ea87a-06c5-4b58-85f1-2cfdad02b59c	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	17:11:54	a772240a-64e0-45d2-9050-92d00ef12f84	column "telefono" of relation "usuarios" contains null values: QueryFailedError	
20231028	17:14:27	ac67bc8e-d882-4d4d-94c2-eaf6032b8cf1	invalid input value for enum usuarios_rol_enum: "0": QueryFailedError	
20231029	13:16:36	7a7d083c-e685-4b1c-91f5-9056ae282b34	column "codigo_de_trabajador" of relation "usuarios" contains null values: QueryFailedError	
20231029	15:09:11	b0227201-e02d-4b50-853c-ca10d891b51f	тип Unable to compile TypeScript:
[96mentities/Producto.ts[0m:[93m5[0m:[93m31[0m - [91merror[0m[90m TS2304: [0mCannot find name 'BaseEntity'.

[7m5[0m export class Producto extends BaseEntity {
[7m [0m [91m                              ~~~~~~~~~~[0m
: TSError	
20231029	15:09:15	9354b1da-5798-4abb-ab67-777bb2503491	тип Unable to compile TypeScript:
[96mentities/Pedido.ts[0m:[93m21[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'BaseEntity'.

[7m21[0m export class Pedido extends BaseEntity {
[7m  [0m [91m                            ~~~~~~~~~~[0m
: TSError	
20231029	15:11:37	114f7f8c-2dd3-4e9c-a5b8-122dbb5f07a1	тип Unable to compile TypeScript:
[96mentities/Pedido.ts[0m:[93m21[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'BaseEntity'.

[7m21[0m export class Pedido extends BaseEntity {
[7m  [0m [91m                            ~~~~~~~~~~[0m
: TSError	
20231029	15:12:21	67a99528-0f77-41d8-a4d5-d9a638ea9cc5	тип Unable to compile TypeScript:
[96mentities/Pedido.ts[0m:[93m21[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'BaseEntity'.

[7m21[0m export class Pedido extends BaseEntity {
[7m  [0m [91m                            ~~~~~~~~~~[0m
: TSError	
20231029	15:13:25	8c705687-144a-4d85-aa9d-fe9d9d4e58f3	тип Unable to compile TypeScript:
[96mentities/Pedido.ts[0m:[93m21[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'BaseEntity'.

[7m21[0m export class Pedido extends BaseEntity {
[7m  [0m [91m                            ~~~~~~~~~~[0m
: TSError	
20231029	15:14:07	b88da446-78ea-4ede-9735-506934b9fc2d	тип Unable to compile TypeScript:
[96mentities/Pedido.ts[0m:[93m21[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'BaseEntity'.

[7m21[0m export class Pedido extends BaseEntity {
[7m  [0m [91m                            ~~~~~~~~~~[0m
: TSError	
20231029	15:14:09	a035da2e-3dbd-429d-895b-785e1aeabb75	тип Unable to compile TypeScript:
[96mentities/Pedido.ts[0m:[93m21[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'BaseEntity'.

[7m21[0m export class Pedido extends BaseEntity {
[7m  [0m [91m                            ~~~~~~~~~~[0m
: TSError	
20231029	15:14:15	9b77c368-be72-4ec9-84d7-dc1de3abbcb5	тип Unable to compile TypeScript:
[96mentities/Pedido.ts[0m:[93m21[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'BaseEntity'.

[7m21[0m export class Pedido extends BaseEntity {
[7m  [0m [91m                            ~~~~~~~~~~[0m
: TSError	
20231029	15:14:51	39908799-566f-46ca-87ba-6873fa110bcd	тип Unable to compile TypeScript:
[96mentities/Producto.ts[0m:[93m5[0m:[93m31[0m - [91merror[0m[90m TS2304: [0mCannot find name 'BaseEntity'.

[7m5[0m export class Producto extends BaseEntity {
[7m [0m [91m                              ~~~~~~~~~~[0m
: TSError	
20231029	15:31:35	1e1c8ac7-e6b8-46d0-adab-314a2dadb5d5	date/time field value out of range: "1698611495162": QueryFailedError	
20231029	15:33:07	f174aa3c-ad43-4b0d-a309-e20745e984ee	date/time field value out of range: "1698611587734": QueryFailedError	
20231029	15:33:41	09b433ce-c3b9-4613-9f0b-a5bec20b07ee	date out of range: "1698611621024": QueryFailedError	
20231029	15:35:34	026286dc-cdea-466d-bbb1-fcfc75eac455	date/time field value out of range: "1698611734820": QueryFailedError	
20231029	15:35:49	d9b3effa-b911-46e5-8023-3a4bf668fb08	date/time field value out of range: "1698611749339": QueryFailedError	
20231029	15:41:38	a33cd667-a321-445c-9515-0ded2a1d0262	date/time field value out of range: "1698612098735": QueryFailedError	
